### PR TITLE
Advanced CARTO.js guide using Airship

### PR DESCRIPTION
**Pull request by jesusbotella**
#241

### CODE REVIEW

File: docs/guides/06-advanced-guide-cartojs.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+```
+
+## Setting a basic layout
+A basic layout might include a container for our map and a sidebar for our future widget.
+
+The layout will always be wrapped by a block element with `.as-app-content` class, which will hold all of our content and elements.
+So, to create our layout we need a right sidebar and a block element wrapped by `.as-map-wrapper` class for our map.
+
+```html
+<section class="as-app-content">
```
Comment (IagoLast) [inline docs/guides/06-advanced-guide-cartojs.md]: `.as-app-content` is normally a `main` tag

File: docs/guides/06-advanced-guide-cartojs.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  </head>
+
+  <body>
```
Comment (IagoLast) [inline docs/guides/06-advanced-guide-cartojs.md]: body should have `as-app` class

File: docs/guides/06-advanced-guide-cartojs.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+client.addLayer(layer);
+client.getLeafletLayer().addTo(map);
+```
+
+## Widget creation
+Widgets are the best way to show data from our visualization. You can create a dashboard with a map and several widgets that will allow you to visualize and explore your data in a clearer and better way.
+
+### Adding a Category Widget
+Adding a Category Widget is as simple as including `<as-category-widget>` tag within our `<body>` tag, setting the options you prefer.
```
Comment (IagoLast) [inline docs/guides/06-advanced-guide-cartojs.md]: to the body or to the sidebar?

File: docs/guides/06-advanced-guide-cartojs.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+      <script src="https://libs.cartocdn.com/carto.js/v4/carto.min.js"></script>
+
+      <style>
+        body {
+          margin: 0;
+          padding: 0;
+        }
+
+        .widgets {
+          padding: 10px;
```
Comment (IagoLast) [inline docs/guides/06-advanced-guide-cartojs.md]: I think we should have a default padding

Comment (ivanmalagon) [reply docs/guides/06-advanced-guide-cartojs.md]: Why can't we use a utility class instead of adding a new class here?

**Comment by jesusbotella** [comment]
I have just updated this PR with the Bounding Box filter :)
The relative data in the widget should be ready whenever we merge the fix :)

**Pull request merged**